python/lsst/pipe/tasks/makeGen3SkyMap.py

```diff
@@ -43,33 +49,16 @@ def validate(self):
                              "skymap to use when writing to the butler")
 
 
-class MakeGen3SkyMapTask(pipeBase.Task):
-    ConfigClass = MakeGen3SkyMapConfig
-    _DefaultName = "makeGen3SkyMap"
-
-    """This is a task to construct and optionally save a SkyMap into a gen3
-    butler repository.
+def makeGen3SkyMap(butler, config):
```

Comment:
@TallJimbo do we want to continue to have Gen3 in the name for this function and the config?

Reply:
Let's drop the "Gen3" everywhere there isn't a direct clash with Gen2.  I think the only place there was a clash before was in the script name, so hopefully that means we can drop the Gen3s everywhere (around this script, at least).